.github/workflows/tests.yml

```diff
@@ -9,14 +9,14 @@ jobs:
         runs-on: ubuntu-latest
         strategy:
             matrix:
-                php: [ 5.6, "7.0", 7.1, 7.2, 7.3, 7.4, "8.0", 8.1 ]
+                php: [ "7.0", 7.1, 7.2, 7.3, 7.4, "8.0", 8.1 ]
```

Comment:
Nit: Any idea why some are wrapped in quotations and others aren't?

Reply:
Yes, it's so the "7.0" isn't converted to "7" when converted to a string. We could wrap them all in quotes, but it's a little more concise this way.

Reply:
That's fine as is, thanks for clarifying.